session.go

```diff
@@ -286,7 +288,11 @@ func (s *Session) keepalive() {
 	for {
 		select {
 		case <-time.After(s.config.KeepAliveInterval):
-			s.Ping()
+			_, err := s.Ping()
+			if err != nil {
+				s.logger.Printf("[ERR] yamux: keepalive failed: %v", err)
+				s.Close()
```

Comment:
It would be good to know if `Close()` returns an error and log it to the console.